packages/web/src/lib/search/Search.ts

```diff
@@ -67,25 +67,33 @@ export class Search {
       adrs.map((adr) => [
         adr.slug,
         {
-          title: adr.title || "Untitled",
-          verbatim: adr.body.enhancedMdx // TODO: remove tags (https://github.com/thomvaill/log4brains/issues/5)
+           title: adr.title || "Untitled",
```

Comment:
Typescript type `AdrForSearch` must be updated too.
You can run `yarn typescript` to ensure your code is valid.
I found out that the typescript checks were indeed not run by the CI pipeline... This is now fixed. Could you please rebase on `master` so the checks are executed?